src/main/java/mekanism/common/item/gear/ItemJetpack.java

```diff
@@ -103,6 +113,11 @@ public void addHUDStrings(List<Component> list, Player player, ItemStack stack,
         }
     }
 
+    @Override
+    public void addCurioHUDStrings(List<Component> list, Player player, ItemStack stack) {
+        addHUDStrings(list, player, stack, getSlot());
+    }
```

Comment:
I feel like this impl should be removed from here and instead moved to `IItemHUDProvider` and have the default be:
```java
if (this instanceof ArmorItem armor) {
    addHUDStrings(list, player, stack, armor.getSlot());
}
```